Dockerfile

```diff
@@ -29,13 +29,13 @@ RUN apt-get update \
         libreadline-dev \
         libsqlite3-dev \
         libtbb-dev \
-        lld-9 \
+        lld \
```

Comment:
Was this reverted on purpose?

Reply:
Something must have gone wrong during merge. I will change it back.